packages/core/src/charts/scatter.ts

```diff
@@ -44,7 +44,9 @@ export class ScatterChart extends AxisChart {
 			new TwoDimensionalAxes(this.model, this.services),
 			new Grid(this.model, this.services),
 			new Ruler(this.model, this.services),
-			new Scatter(this.model, this.services),
+			new Scatter(this.model, this.services, {
+				ignoreScatterDisabled: true
```

Comment:
we probably need a more readable name here. If I was looking at this file for the first time I probably wouldn't understand what this flag would do

Reply:
how about `alwaysEnableScatterDot`?